src/ansi-c/ansi_c_language.h

```diff
@@ -22,13 +22,30 @@ Author: Daniel Kroening, kroening@kroening.com
 // clang-format off
 #define OPT_ANSI_C_LANGUAGE \
   "(max-nondet-tree-depth):" \
-  "(min-null-tree-depth):"
+  "(min-null-tree-depth):" \
+  "(pointers-to-treat-as-array):" \
```

Comment:
`pointers-to-treat-as-array` -> `pointers-to-treat-as-arrays` (and throughout the PR)